src/models/index.js

```diff
@@ -1,31 +1,56 @@
-'use strict';
+/* eslint-disable no-console */
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+import configJson from '../config/config';
 
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
 const basename = path.basename(__filename);
-const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../config/config.js')[env];
+const env = process.env.NODE_ENV ? process.env.NODE_ENV : 'development';
+
+const config = configJson[env];
+
+console.log('current environment is: ', env);
+
 const db = {};
 
 let sequelize;
-if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+if (config.environment === 'production') {
+  sequelize = new Sequelize(
+    process.env.DB_NAME,
+    process.env.DB_USERNAME,
+    process.env.DB_PASSWORD,
+    {
+      host: process.env.DB_HOST,
+      port: process.env.DB_PORT,
+      dialect: 'postgres',
+      dialectOption: {
+        ssl: true,
+        native: true,
+      },
+      logging: true,
+    }
+  );
```

Comment:
Why the change from what was auto generated by sequelise?

Reply:
what was auto generated was not in ES6